src/boundary/IMemberTask.java

```diff
@@ -0,0 +1,7 @@
+package boundary;
+
+import entity.MemberTask;
+
+public interface IMemberTask {
```

Comment:
Boundaries are "Roles". IMemberTask is not a Role (i.e. Actor) that uses the system. MemberTask is an "Entity" that is used by the MemberInteractor. viewTask belongs with its actor which is IMember. A member is the actor that will use viewTask, thus it belongs in the member boundary.

Reply:
Seen. Thank you for the clarification.